### PR TITLE
WIP: Writing tests and fixing bugs in Trainer, Profile, Certs components and services

### PR DESCRIPTION
**Pull request by emily-v**
Components/Services being worked on:

trainers component
trainer-item component
trainer-add component
add-trainer-error component
certifications component
profile component
filehandler-controller service
trainer-controller service

### CODE REVIEW

File: src/app/components/profile/profile.component.ts [hunk 1]
```diff
@@ -186,15 +188,7 @@ export class ProfileComponent implements OnInit {
     } else return number;
   }
 }
-/*
```
Comment (juliozev) [inline src/app/components/profile/profile.component.ts]: woah....you can't just get rid off Geoffrey the giraffe

Comment (emily-v) [reply src/app/components/profile/profile.component.ts]: he'll always live on in the history

Comment (ajduet) [reply src/app/components/profile/profile.component.ts]: @juliozev she killt it good.

Comment (juliozev) [reply src/app/components/profile/profile.component.ts]: @ajduet 
image

**Pull request merged**